src/containers/contact/Contact.js

```diff
@@ -7,6 +7,9 @@ import StyleContext from "../../contexts/StyleContext";
 
 export default function Contact() {
   const { isDark } = useContext(StyleContext);
+  if (!contactInfo.display) {
```

Comment:
We don't have this property in `contactInfo`, I don't think we should extend to include this either because contact information is pretty essential for a portfolio.